docs/backends/linux/index.rst

```diff
@@ -3,42 +3,55 @@
 Linux
 =====
 
-...
+.. _sec-backends-linux-configuration:
 
-Device Identifiers
-------------------
+System Configuration
+--------------------
+
+To use the Linux backend you need to be running a Linux distribution. We
+support a number of Linux distributions, we recommend using one of the
```

Comment:
without a conjunction after the comma, it sounds a bit weird ... for macOS and freebsd, you write "..., however to ensure capability ..."

Reply:
I changed the wording, it should be better now